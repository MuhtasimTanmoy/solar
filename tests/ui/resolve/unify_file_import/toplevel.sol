// OK
import "./auxiliary/file1.sol";
import "./auxiliary/file2.sol";
import "./auxiliary/imported.sol" as Imported;
import "./auxiliary/imported.sol" as Imported;

//~v ERROR: identifier `Imported` already declared
import "./auxiliary/different_imported.sol" as Imported;
