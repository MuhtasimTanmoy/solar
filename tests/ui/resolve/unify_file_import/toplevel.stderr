error: identifier `Imported` already declared
  --> ROOT/tests/ui/resolve/unify_file_import/toplevel.sol:LL:CC
   |
LL | import "./auxiliary/file1.sol";
   | ------------------------------- note: previous declaration declared here
LL | import "./auxiliary/file2.sol";
...
LL |
LL | import "./auxiliary/different_imported.sol" as Imported;
   |                                                ^^^^^^^^
   |

error: aborting due to 1 previous error

